app/main/views/marketplace.py

```diff
@@ -83,6 +83,13 @@ def index():
     )
 
 
+@main.route('/_test_harakiri', methods=['GET'])
+def test_timeout():
+    from time import sleep
```

Comment:
Can we put imports at the top? I can't remember if this explicitly fails `flake8` or not...